samza-core/src/main/java/org/apache/samza/operators/MessageStreamImpl.java

```diff
@@ -65,51 +65,55 @@
    *
    * @param graph the {@link StreamGraphImpl} object that this stream belongs to
    */
-  MessageStreamImpl(StreamGraphImpl graph) {
+  public MessageStreamImpl(StreamGraphImpl graph) {
```

Comment:
It's currently used in the "streams" sub-package that contains Input/Intermediate/OutputStreamImpl classes (didn't want to leave them all in the top-level package). I'll change it back if I end up switching to Input/OutputOperatorSpecs.